projects/storefrontapp-e2e-cypress/cypress/integration/added-to-cart-modal-e2e-spec.ts

```diff
@@ -0,0 +1,182 @@
+const productId = '3595723';
+const productId2 = '3325048';
+
+describe('Added to cart modal', () => {
+  before(() => {
+    cy.visit(`/product/${productId}`);
+  });
+
+  it('basic modal behavior', () => {
+    // increase the quantity to 2 and add it to cart
+    cy.get('cx-product-summary .cx-item-counter.btn-group button')
+      .contains('+')
+      .click();
+
+    cy.get('cx-product-summary cx-add-to-cart button').click();
+
+    // quantity is set correctly
```

Comment:
This comment is confusing. it first check product name, then check quantity. We better check price as well.